R/format-table.R

```diff
@@ -11,7 +11,8 @@ source(file.path(getwd(), "R", "constants.R"))
 format_table <-
   function(input_table,
            columns_to_format = NULL,
-           remove_duplicate = FALSE) {
+           remove_duplicate = FALSE,
```

Comment:
Can you documentation for two params `remove_duplicate` and `strip_invalid_part_ID`.